xen-bugtool

```diff
@@ -1003,6 +1005,9 @@ exclude those logs from the archive.
     file_output(CAP_XENSERVER_CONFIG, [SYSCONFIG_CLOCK])
     tree_output(CAP_XENSERVER_CONFIG, SYSTEMD_CONF_DIR)
     file_output(CAP_XENSERVER_CONFIG, [CGRULES_CONF])
```

Comment:
Presume the config file/directory is not large, so we do not need to enlarge `CAP_XENSERVER_CONFIG`
also presume the security level is conform to `CAP_XENSERVER_CONFIG`